### PR TITLE
[MMCA - 4450] - Remove unused code and ToDos through out the services 

### PR DESCRIPTION
**Pull request by tyagikumarvineet**
Description - Code changes to prepare repo to hand over to Live services

Below has been covered as part of this PR

- removed unused code
- remove descriptive comments
- add scalastyle config for code and tests
- replace hard coded values with the constants
- fix long lines
- fix compilation warnings
- add Consants file
- add CommonTestData file in tests
- update sbt-auto-build to 3.20.0
- refactoring

Note: - Logic should be removed from below views as part of separate tickets as that needs considerable time and testing
              1. import_vat.scala.html
              2. security_statements.scala.html
              3. postponed_import_vat.scala.html

### CODE REVIEW

File: app/viewmodels/PostponedVatViewModel.scala [hunk 1]
```diff
@@ -21,11 +21,17 @@ import play.api.i18n.Messages
...
-  def apply(files: Seq[PostponedVatStatementFile])(implicit messages: Messages, dateTimeService: DateTimeService): Seq[PostponedVatStatementGroup] = {
-    val response: Seq[PostponedVatStatementGroup] = files.groupBy(_.monthAndYear).map { case (month, filesForMonth) => PostponedVatStatementGroup(month, filesForMonth) }.toList
+  def apply(files: Seq[PostponedVatStatementFile])(implicit messages: Messages,
+                                                   dateTimeService: DateTimeService): Seq[PostponedVatStatementGroup] = {
+    val response: Seq[PostponedVatStatementGroup] =
+      files.groupBy(_.monthAndYear).map {
+        case (month, filesForMonth) => PostponedVatStatementGroup(month, filesForMonth)
+      }.toList
+
     val monthList = (1 to 6).map(n => dateTimeService.systemDateTime().toLocalDate.minusMonths(n))
```
Comment (AdamCoops) [inline app/viewmodels/PostponedVatViewModel.scala:31]: Do we want to remove these hard coded nums? I don't think values below 10 flag as magic number warnings in scalastyle

File: app/views/Layout.scala.html [hunk 2]
```diff
@@ -31,33 +31,35 @@
         hmrcTimeoutDialog: HmrcTimeoutDialog,
         h2: components.h2,
         p: components.p
-
 )
 
-@(pageTitle: Option[String] = None,
+@(
+        pageTitle: Option[String] = None,
```
Comment (AdamCoops) [inline app/views/Layout.scala.html:37]: is this over indented?

File: test/connectors/SdesConnectorSpec.scala [hunk 3]
```diff
@@ -151,103 +195,370 @@ class SdesConnectorSpec extends SpecBase {
...
+    val sdesSecurityStatementsUrl =
+      "http://localhost:9754/customs-financials-sdes-stub/files-available/list/SecurityStatement"
+
+    val year2018 = 2018
+    val month3 = 3
+    val month4 = 4
+    val month5 = 5
+    val month6 = 6
+
+    val startMonth3 = 3
```
Comment (AdamCoops) [inline test/connectors/SdesConnectorSpec.scala:219]: We can use the CommonTestData values here like CommonTestData.MONTH_3, etc

File: app/viewmodels/PostponedVatViewModel.scala (hunk 1 above)
Comment (tyagikumarvineet) [reply app/viewmodels/PostponedVatViewModel.scala]: Done. Have created a constant named MONTHS_RANGE_ONE_TO_SIX_INCLUSIVE

File: app/views/Layout.scala.html (hunk 2 above)
Comment (tyagikumarvineet) [reply app/views/Layout.scala.html]: Done. It was aligned with parameters of @this(. But now, both have be done and reduced one extra tab

**Comment by AdamCoops** [comment]
Line 48 and 87 of SecuritiesController can be deleted

**Comment by tyagikumarvineet** [comment]
> Line 48 and 87 of SecuritiesController can be deleted

Done. I have deleted line 87 as well. But we can go with either no line before last } or with a line break.

**Review by AdamCoops — APPROVED** [review]

**Pull request merged**